scripts/port_clash_fixup.py

```diff
@@ -0,0 +1,84 @@
+#!/usr/bin/env python3
+# Copyright 2020 ETH Zurich
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""
+:mod: scripts.port_clash_fixup
+=========================
+
+Prints existing port clashes and fixes them.
+For every clash, the interface that was modified the first is kept, but
+for all the other ones a new port is found, and the associated
+AP's configuration version is bumped.
+This is intended to be used only once.
+
+Run with python manage.py runscript port_clash_fixup
```

Comment:
This is one of the things I noticed during the upgrade procedure; `runscript` is currently not available in the production environment.
Please add it, by moving `django_extensions` from the `INSTALLED_APPS` setting in `scionlab/settings/development.py` to `scionlab/settings/common.py`.